docs/recipes/ui/add-bulma.md

```diff
@@ -0,0 +1,32 @@
+# How do I add Bulma to a SAFE project?
+
+[Bulma](https://bulma.io/documentation/) is a free open-source UI framework based on flex-box that helps you create modern and responsive layouts.
+
+When using Feliz (the standard for a SAFE app), follow the instructions below. When using Fable.React, use the [Fulma](https://fulma.github.io/Fulma/) wrapper for Bulma.
+
+## 1. Add the Feliz.Bulma NuGet package to the solution
+
+```
+dotnet paket add Feliz.Bulma -p Client
```

Comment:
This command will only work if executed from within `./src` right (let me know if I'm wrong)? If the location to execute a command from is not specified, I would assume the repo root.

```suggestion
dotnet paket add Feliz.Bulma -p src/Client
```

Reply:
For me, specifying projects by name instead of path works. I'd say it's cleaner that way right?

Reply:
Even from the project root by the way